opencodelists/settings.py

```diff
@@ -39,11 +39,11 @@
 SECRET_KEY = env.str("SECRET_KEY")
 
 # SECURITY WARNING: don't run with debug turned on in production!
```

Comment:
Are these settings changed deliberate?

Reply:
sorry - that was not deliberate. I added a file that I was playing around with!